english/polar-bear-attack/polar-bear-attack.md~

```diff
@@ -0,0 +1,7 @@
+You leap and jump on the back of the polar bear. It bucks and tries to throw
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.